packages/cashier/src/components/crypto-fiat-converter/crypto-fiat-converter.tsx

```diff
@@ -154,9 +153,8 @@ const CryptoFiatConverter = observer(
                                     setArrowIconDirection('left');
                                 }}
                                 onChange={(e: TReactChangeEvent) => {
-                                    const from_rate = getRate(from_currency || '');
-                                    const to_rate = getRate(to_currency || '');
-                                    const converted_amount = (Number(e.target.value) * from_rate) / to_rate;
+                                    const rate = exchange_rates[from_currency][to_currency];
```

Comment:
better to calculate the rate outside the rendering part so you can prevent duplication(line 118)

Reply:
@mahdiyeh-deriv so I need to store it in a new state and keep it updated by a useEffect right?

Reply:
done ✅